Content.Server/Xenoarchaeology/XenoArtifacts/Events/ArtifactActivatedEvent.cs

```diff
@@ -0,0 +1,12 @@
+using Robust.Shared.GameObjects;
+
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Events;
+
+/// <summary>
+///     Invokes when artifact was successfully activated.
+///     Used to start attached effects.
+/// </summary>
+public class ArtifactActivatedEvent : EntityEventArgs
+{
+    public EntityUid? User;
```

Comment:
maybe rename to "ActivatingEntity" or something since "user" implies it'll always be a player